tests/publiccloud/prepare_instance.pm

```diff
@@ -14,6 +14,7 @@ use publiccloud::ssh_interactive "select_host_console";
 use testapi;
 use version_utils;
 use utils;
+use publiccloud::utils;
```

Comment:
I believe the changes in this file might have been carried over by mistake

Reply:
Should I delete this line?

Reply:
No need, but ideally the following should fix this : 
* git checkout master
* git pull upstream master
* git checkout default_yaml
* git rebase master
* git push -f origin default_yaml

On the above, upstream is github.com/os-autoinst/os-autoinst-distri-opensuse
origin is github.com/skliu/os-autoinst-distri-opensuse

Reply:
if you use different names for the origin and upstream that I mentioned above, just adapt accordingly. You can do `git remote -v` to check which repo name corresponds to which url

Reply:
Done, thank you for the detailed steps.